README.md

```diff
@@ -276,13 +276,17 @@ This case is however handled by `::PDI_multi_expose` call that exposes all data,
 then triggers an event and finally does all the reclaim in reverse order.
 
 * Replace the remaining `::PDI_share`/`::PDI_reclaim` by `::PDI_expose`s and
-  `::PDI_multi_expose`s and ensure that your code keeps the exact same behaviour
-  by comparing its trace to `ex6.log` (only the lines matching `[Trace-plugin]`
-  have been kept). You can easily check if the files are the same by running:
+  `::PDI_multi_expose`s and ensure that your code keeps the exact same behaviour as in previous exercise by comparing its trace to `ex6.log`
+  (only the lines matching `[Trace-plugin]` have been kept). Using the previous section [Execution with storage of the log](#execution-with-storage-of-the-log),
+  run  this exercise in saving the output log in the `ex6.result.log`.
+  After that you can easily check if the files are the same by running:
 ```bash
   diff ex6.log <(grep Trace-plugin ex6.result.log)
 ```
 
+In summary:\n 
+  1. `::PDI_expose` is equivalent to `::PDI_share` + `::PDI_reclaim` \n 
+  2. `::PDI_multi_expose` is equivalent to `::PDI_share` + `::PDI_event` + `::PDI_reclaim`\n 
```

Comment:
"\n" is not a line break in Markdown.
"[...] differs significantly from most other text-to-HTML formatters [...] you end a line with two or more spaces, then type return."
```suggestion
In summary:  
  1. `::PDI_expose` is equivalent to `::PDI_share` + `::PDI_reclaim`  
  2. `::PDI_multi_expose` is equivalent to `::PDI_share` + `::PDI_event` + `::PDI_reclaim`  
```